docker/tests/runLinter.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+set -e -x
+vale .
```

Comment:
same here, I think this can go into a `Makefile` in the maistra.github.io repo

Reply:
Thanks. Fixed in https://github.com/Maistra/maistra.github.io/pull/123